Library/Formula/puddletag.rb

```diff
@@ -41,6 +41,12 @@ def install
     end
 
     cp_r buildpath/"source/.", buildpath
+    inreplace buildpath/"puddlestuff/puddleobjects.py" do |s|
+      s.gsub! /QPalette.Window/, "pal.window()"
+    end
+    inreplace buildpath/"puddlestuff/tagmodel.py" do |s|
+      s.gsub! /amarok -p/, "open -a iTunes"
```

Comment:
You should raise this issue to upstream; they could use the correct command depending on the platform (`from sys import platform` then test if e.g. `platform == "darwin"` then use iTunes).